NOTICES.txt

```diff
@@ -360,200 +500,873 @@ TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
-          instructions on where to get the original software distribution.
+--------------- SECTION 2: BSD 3-clause "New" or "Revised" License ----------
 
-       d) make other distribution arrangements with the author.
+BSD 3-clause "New" or "Revised" License is applicable to the following component(s).
+
+>>> https://rubygems.org/gems/puma/versions/3.12.2
+
+BSD 3-Clause License
+
+Copyright (c) 2019, Evan Phoenix. Some code by Zed Shaw, (c) 2005.
+All rights reserved.
 
-  3. You may distribute the software in object code or binary form,
-     provided that you do at least ONE of the following:
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
 
-       a) distribute the binaries and library files of the software,
-	  together with instructions (in the manual page or equivalent)
-	  on where to get the original distribution.
+1. Redistributions of source code must retain the above copyright notice, this
+   list of conditions and the following disclaimer.
 
-       b) accompany the distribution with the machine-readable source of
-	  the software.
+2. Redistributions in binary form must reproduce the above copyright notice,
+   this list of conditions and the following disclaimer in the documentation
+   and/or other materials provided with the distribution.
 
-       c) give non-standard binaries non-standard names, with
-          instructions on where to get the original software distribution.
+3. Neither the name of the copyright holder nor the names of its
+   contributors may be used to endorse or promote products derived from
+   this software without specific prior written permission.
 
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+>>> https://rubygems.org/gems/pg/versions/0.20.0
+
+Ruby is copyrighted free software by Yukihiro Matsumoto <matz@netlab.jp>.You can redistribute it and/or modify it under either the terms of the2-clause BSDL (see the file BSDL), or the conditions below:
+  1. You may make and give away verbatim copies of the source form of the     software without restriction, provided that you duplicate all of the     original copyright notices and associated disclaimers.
```

Comment:
It looks like something is wrong with the formatting.  It's coming out as if it's in two columns.  Let's try keep it in the format listed above (single column).